docs/pages/dev/reference/API/org_unit_change_request_configuration.md

```diff
@@ -0,0 +1,240 @@
+- One or more `editable_fields` value is unknown
+- One or more `possible_parent_types` is not a suitable for as a parent for the given OrgUnit Type
+- One or more `editable_reference_forms` is not a reference form for the given OrgUnit Type
+
+#### 401 - Unauthorized
+
+- No authentication token or an invalid one was provided
+
+#### 403 - Forbidden
+
+- User doesn't have the proper permission
+
+#### 404 - Not found
+
+- `project_id` is not null and doesn't exist
+- `org_unit_type_id` is not null and doesn't exist
+- One or more `possible_parent_types` doesn't exist
+- One or more `possible_group_sets` doesn't exist
+- One or more `editable_reference_forms` doesn't exist
+
+
+## 2 - GET /api/change_requests_configs/
+
+API used to list all Change Request configurations.
+
+### Permissions
+
+- Change Request Configuration
+
+### Query parameters
+- page: Int (optional) - Current page (default: 1)
+- limit: Int (optional) - Number of entities returned by page (default: 20)
+- org_unit_type_id: Int (optional) - Id of the OrgUnitType to filter on
+- project_id: Int (optional) - Id of the Project to filter on
+
+### Possible responses
+
+#### 200 - OK
+
+```json
+{
+  "count": "Long",
+  "has_next": "Boolean",
+  "has_previous": "Boolean",
+  "page": "Long",
+  "pages": "Long",
+  "limit": "Long",
+  "results": [
+    {
+      "id": "Int - id in the database, I know Martin wants it",
+      "project_id": "Int - Project ID",
+      "project_name": "String - Project name",
+      "org_units_editable": "Boolean - Whether or not OrgUnits of this OrgUnit Type are editable",
+      "org_unit_type_id": "Int - OrgUnit Type ID",
+      "org_unit_type_name": "String - OrgUnit Type name",
+      "editable_fields": "Array<String> - List of possible fields",
+      "possible_parent_types": [
+        {
+        "id": "Int - OrgUnit Type ID",
+        "name": "String - OrgUnit Type name"
+        }
+      ],
+      "possible_group_sets": [
+        {
+          "id": "Int - Group ID",
+          "name": "String - Group name"
+        }
+      ],
+      "editable_reference_forms": [
+        {
+          "id": "Int - Form ID",
+          "name": "String - Form name"
+        }
+      ],
+      "created_at": "Timestamp",
+      "updated_at": "Timestamp"
+    }
+  ]
+}
+```
+
+#### 400 - Bad request
+
+- One or more of the parameters provided couldn't be parsed
+- One or more of the parameters provided couldn't matched to the related resource (E.g.: unknown project_id)
+- A parameter was given that is not recognized
+
+#### 401 - Unauthorized
+
+- No authentication token or an invalid one was provided
+
+#### 403 - Forbidden
+
+- User doesn't have the proper permission to access this resource.
+
+
+
+## 3 - GET /api/change_requests_configs/{id}/
+
+API used to retrieve a Change Request configuration.
+
+### Permissions
+
+- Change Request Configuration
+
+### Possible responses
+
+#### 200 - OK
+
+```json
+{
```

Comment:
@bmonjoie both the `list` and the `retrieve` endpoints return the same data, is there a reason for that? Shouldn't we restrict the `list` to the main fields and send the whole thing in `retrieve`?

Reply:
The list seems to display almost everything, so it's easier to use the same `Serializer`, I guess. But it was a proposition; it should probably be discussed with @beygorghor what he wants for the frontend web